### ISSUE TITLE
`pac auth name` outputs an incorrect error message if name > 30 characters

### ISSUE THREAD

**Issue by filcole** [microsoft/powerplatform-vscode#197]
If a profile is renamed using `pac auth name` with a new name longer than 30 characters, the error message still says a maximum length of 12 characters, even though the maximum length is now 30 characters.

To reproduce try to rename an existing profile with a name with 31 characters, e.g.

```
PS C:\Dev\XXXX> pac auth name --index 10 --name 1234567890123456789012345678901
Microsoft PowerPlatform CLI
Version: 1.15.3+g79950a8

Error: Argument --name value is incorrect: The length of the name should be between 1 and 12.

Usage: pac auth name --index --name

  --index                     The index of the profile to be named/renamed (alias: -i)
  --name                      The name you want to give to this authentication profile (maximum 12 characters) (alias: -n)
```

The error message should say between 1 and 30, and not 1 and 12.

**Comment by tehcrashxor** [comment]
Tracking as AB#2724005

**Comment by tehcrashxor** [comment]
Fix has been merged into the PAC codebase and should make it into the next release.

**Comment by tehcrashxor** [comment]
The fix has been rolled out with PAC v1.16.6 / powerplatform-vscode v1.0.16 in #216
